homeassistant/components/nest/strings.json

```diff
@@ -26,12 +26,26 @@
         "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
       },
       "pubsub": {
-        "title": "Configure Google Cloud",
-        "description": "Visit the [Cloud Console]({url}) to find your Google Cloud Project ID.",
+        "title": "Configure Google Cloud Pub/Sub",
+        "description": "Home Assistant uses Cloud Pub/Sub receive realtime Nest device updates. Nest servers publish updates to a Pub/Sub Topic and Home Assistat receives the updates through a Pub/Sub Subscription.\n\n1. Visit the [Device Access Console]({device_access_console_url}) and ensure a Pub/Sub topic is configured.\n2. Visit the [Cloud Console]({url}) to find your Google Cloud Project ID and confirm it is correct below.\n3. The next step will attempt to audo-discover Pub/Sub topics and subscriptions.\n\nSee the integration documentation for [more info]({more_info_url}).",
         "data": {
           "cloud_project_id": "[%key:component::nest::config::step::cloud_project::data::cloud_project_id%]"
         }
       },
+      "pubsub_topic": {
+        "title": "Configure Cloud Pub/Sub Topic",
+        "description": "Nest devices publish updates on a Cloud Pub/Sub Topic. Select the Pub/Sub topic below that is the same as the [Device Access Console]({device_access_console_url}). See the integration documentation for [more info]({more_info_url}).",
+        "data": {
+          "topic_name": "Pub/Sub Topic Name"
```

Comment:
is `Topic` a brand name or is it just a normal word, otherwise we should lowercase that word

Reply:
Lowercasing.